server/app/models/List.js

```diff
@@ -6,32 +6,38 @@ const options = {
   timestamps: true,
 };
 
-const ListSchema = mongoose.Schema({
-  name: {
-    type: String,
-    required: [true, 'Please enter a list name'],
-  },
-  coverUrl: {
-    type: String,
-    required: [true, 'Please add a cover URL'],
-  },
-  user: {
-    type: mongoose.Schema.ObjectId,
-    ref: 'User',
-    required: true,
-  },
-  products: {
-    type: [mongoose.Schema.ObjectId],
-    ref: 'Product',
+const ListSchema = mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: [true, 'Please enter a list name'],
+    },
+    coverUrl: {
+      type: String,
+      required: [true, 'Please add a cover URL'],
```

Comment:
Maybe this shouldn't be required. We could have a default image for an empty list, and if there's no coverUrl, the frontend could display that image or the image of the first product on that list. What do you guys think, @Esodey and @vic2019 ?

Reply:
That makes sense to me